.changeset/soft-owls-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ea-bootstrap': minor
```

Comment:
I'm tempted to say this should be a `patch`. Any arguments for why it should be `minor`?

Reply:
done!